Slim/Player/SqueezePlay.pm

```diff
@@ -89,6 +91,7 @@ my %CapabilitiesMap = (
 	Spdirect                => 'spDirectHandlers',
 	Proxy                   => 'proxyAddress',
 	CanHTTPS                => '_canHTTPS',
+	Balance                 => 'balance',
```

Comment:
Just wanted to give this a try... but which player would support balance?

Reply:
I just pushed an update to squeezelite-esp32